### PR TITLE
Add switch preference for clearing browsing data

### PR DESCRIPTION
**Pull request by codeimpl**
Add switch preference to delete browsing data on exit.

Refs: #768

### CODE REVIEW

File: res/xml/banana_extension_preferences.xml [hunk 1]
```diff
@@ -73,6 +73,11 @@
 
     <PreferenceCategory
         android:title="@string/user_data"/>
+    <org.chromium.components.browser_ui.settings.ChromeSwitchPreference
+        android:key="feature_name_clear_browsing_data"
+        android:title="@string/clear_browsing_data_title"
```
Comment (romandev) [inline res/xml/banana_extension_preferences.xml:78]: auto_clear_browsing_data_title

File: res/xml/banana_extension_preferences.xml [hunk 2]
```diff
@@ -73,6 +73,11 @@
...
+        android:summaryOn="@string/clear_browsing_data_summary"
```
Comment (romandev) [inline res/xml/banana_extension_preferences.xml:79]: auto_clear_browsing_data_summary

File: res/xml/banana_extension_preferences.xml [hunk 3]
```diff
@@ -73,6 +73,11 @@
 
     <PreferenceCategory
         android:title="@string/user_data"/>
+    <org.chromium.components.browser_ui.settings.ChromeSwitchPreference
```
Comment (romandev) [inline res/xml/banana_extension_preferences.xml:76]: Move this option bottom of "clear_browsing_data"

File: src/org/triple/banana/settings/ExtensionFeatures.java [hunk 4]
```diff
@@ -196,6 +203,7 @@ public void onSecurityLevelChanged(SecurityLevel newLevel) {
         // upstream. For sync with this value in triple banana, BOTTOM_TOOLBAR feature name has
         // diffrent name style like others.
         public static final String BOTTOM_TOOLBAR = "bottom_toolbar_enabled";
+        public static final String CLEAR_BROWSING_DATA = "feature_name_clear_browsing_data";
```
Comment (romandev) [inline src/org/triple/banana/settings/ExtensionFeatures.java:206]: AUTO_CLEAR_BROWSING_DATA = "feature_name_auto_clear_browsing_data"

File: src/org/triple/banana/settings/ExtensionFeatures.java [hunk 5]
```diff
@@ -196,6 +203,8 @@ public void onSecurityLevelChanged(SecurityLevel newLevel) {
         // upstream. For sync with this value in triple banana, BOTTOM_TOOLBAR feature name has
         // diffrent name style like others.
         public static final String BOTTOM_TOOLBAR = "bottom_toolbar_enabled";
+        public static final String AUTO_CLEAR_BROWSING_DATA =
```
Comment (romandev) [inline src/org/triple/banana/settings/ExtensionFeatures.java:206]: Please keep alhabetical order

File: res/values-ko/strings.xml [hunk 6]
```diff
@@ -18,6 +18,7 @@
     <string name="browser_lock">브라우저 잠금</string>
     <string name="browser_lock_description">브라우저 잠금을 해제하기 위해\n생체 인증을 진행하세요</string>
     <string name="browser_lock_summary">브라우저를 열 때, 추가 인증을 사용하여 개인정보를 안전하게 보호하세요. 사용자 편의를 위해 인증 성공 후 1분 간은 세션이 유지됩니다.</string>
+    <string name="clear_browsing_data_summary">브라우저 종료시 인터넷 사용 기록, 쿠키 및 사이트 데이터, 캐시된 이미지 및 파일들을 삭제합니다.</string>
```
[the remark this answers is not in the document]
Comment (codeimpl) [reply res/values-ko/strings.xml]: done

File: res/values/strings.xml [hunk 7]
```diff
@@ -18,6 +18,7 @@
     <string name="browser_lock">Browser Lock</string>
     <string name="browser_lock_description">Scan your biometric data to unlock browser</string>
     <string name="browser_lock_summary">Use additional authentications to protect your personal data securely when opening Banana Browser. Once the authentication is success, its authenticated session is kept for 1 minute for your convenience.</string>
+    <string name="clear_browsing_data_summary">Browsing history, cookies and site data, cached image and files are deleted, When you exit browser.</string>
```
[the remark this answers is not in the document]
Comment (codeimpl) [reply res/values/strings.xml]: done

File: res/xml/banana_extension_preferences.xml (hunk 1 above)
Comment (codeimpl) [reply res/xml/banana_extension_preferences.xml]: done

File: res/xml/banana_extension_preferences.xml (hunk 2 above)
Comment (codeimpl) [reply res/xml/banana_extension_preferences.xml]: done

File: res/xml/banana_extension_preferences.xml (hunk 3 above)
Comment (codeimpl) [reply res/xml/banana_extension_preferences.xml]: done

File: src/org/triple/banana/settings/ExtensionFeatures.java (hunk 4 above)
Comment (codeimpl) [reply src/org/triple/banana/settings/ExtensionFeatures.java]: done

File: src/org/triple/banana/settings/ExtensionFeatures.java (hunk 5 above)
Comment (codeimpl) [reply src/org/triple/banana/settings/ExtensionFeatures.java]: done

**Pull request merged**